.github/workflows/test.yml

```
@@ -36,6 +36,7 @@ jobs:
           - 3.9
           - '3.10'  # Needs quotes so YAML doesn't think it's 3.1
           - '3.11'
+          - '3.12.0-rc.3'
```

according to https://github.com/actions/setup-python/issues/736 3.12 was released so I guess we can just use it

```suggestion
          - '3.12'
```

---

ha -- forgot to commit... well -- will do locally since anyways need to add more ignores